plugin-core/src/main/java/com/salesforce/bazel/eclipse/BazelNature.java

```diff
@@ -47,12 +47,6 @@
     public static final String BAZEL_NATURE_ID = "com.salesforce.bazel.eclipse.bazelNature"; //$NON-NLS-1$
     private IProject project;
     
-    /**
-     * We create a special Eclipse project to represent the Bazel workspace. This is the base name of the project.
-     * The actual name of the workspace is added to this string, like "Bazel Workspace (acme)"
-     */
-    public static final String BAZELWORKSPACE_PROJECT_BASENAME = "Bazel Workspace";
```

Comment:
I moved most code that uses this constant into a centralized place, and removed it from here because it isn't related to the BazelNature.